app/sidekiq/central_mail/submit_central_form686c_job.rb

```diff
@@ -24,6 +25,17 @@ def extract_uuid_from_central_mail_message(data)
       data.body[/(?<=\[).*?(?=\])/].split(': ').last if data.body.present?
     end
 
+    RETRY = 14
+
+    sidekiq_options retry: RETRY
+
+    sidekiq_retries_exhausted do |msg, _ex|
+      Rails.logger.error(
+        "Failed all retries on CentralMail::SubmitCentralForm686cJob, last error: #{msg['error_message']}"
```

Comment:
can you move last error into a param?

something like:
```
    Rails.logger.error(
        'Failed all retries on CentralMail::SubmitCentralForm686cJob.',
        { claim_id: @claim.id, lastError: msg['error_message'] }
      )
```

this makes it a lot easier to search in datadog